lib/bolt/bolt_option_parser.rb

```diff
@@ -6,6 +6,21 @@
 
 module Bolt
   class BoltOptionParser < OptionParser
+    INCLUDE_OPTS = { 'apply' => [:common_opts, 'noop', 'execute', 'compile-concurrency'],
```

Comment:
We should just build these lists up by combining the lists defined in OPTIONS rather then using magic keyword symbols that get looked up later.

ie define `OPTIONS` then look up sets of options from it to build the options for each command. `'apply' => OPTIONS[:run_opts] + ['noop', 'execute', 'compile-concurrency']`

It may be easier to do this if you build this list in a method rather then with constants.

calling the `common_opts` run_opts or `action_opts` is probably clearer.